addons/game/XEH_pre_init.sqf

```diff
@@ -240,6 +240,9 @@ FUNC(getGear) = {
     if (isNull _unit) exitWith {[]};
     // diag_log text format["Assigned Items: %1", (assignedItems _unit)];
     private _gear = (weapons _unit) + (items _unit) + (assignedItems _unit);
+    _gear = _gear select {(_x select [0, 4]) == "ACRE" || _x == "ItemRadio" || _x == "ItemRadioAcreFlagged"}; // We are only interested in ACRE gear.
```

Comment:
Indeed, even without lazy eval here. I didn't lazy eval to improve worst case performance (lazy eval adds some overhead). Though the monitorRadios function (client check for uninitalized radios and a few other things) would do 3 config lookups for each equipment return. But filtering in getGear seems to speed it up. Each in the server masterIDTracker there is a speed up and that's only 1 config look up.